localdev/run.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+INFRA_SVCS="redis postgres pulsar stan"
```

Comment:
Do we want this controlled from ENV vars or something? I feel like we want users to not have to edit scripts in the repo to stop stuff being accidentally commiteed

Reply:
not sure what you mean?

Reply:
Just to check, these variables determine what components are run with docker-compose?

If a user decides they want to change it, and then writes a commit, those changes to this script will be sent in that PR. Ideally, we don't want these scripts to need any user-edits, and instead control them from enviromental variables / a config file that is local to the user. Does this make sense?